deploy/helm-chart/README.md

```diff
@@ -293,6 +293,9 @@ The following tables lists the configurable parameters of the NGINX Gateway Fabr
 | `nginxGateway.replicaCount`                       | The number of replicas of the NGINX Gateway Fabric Deployment.                                                                                                                                           | 1                                                                                                               |
 | `nginxGateway.leaderElection.enable`              | Enable leader election. Leader election is used to avoid multiple replicas of the NGINX Gateway Fabric reporting the status of the Gateway API resources.                                                | true                                                                                                            |
 | `nginxGateway.leaderElection.lockName`            | The name of the leader election lock. A Lease object with this name will be created in the same Namespace as the controller.                                                                             | Autogenerated                                                                                                   |
+| `nginxGateway.securityContext.allowPrivilegeEscalation` | Some environments may need this set to true in order for the control plane to successfully reload NGINX. | false |
```

Comment:
2 out of 3 helm parameters come out of the blue and not related to the feature of the PR.
Are we adding them because they have not been added before?